site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -765,7 +765,17 @@ components:
                type: array
                items:
                   type: string
-               description: The device IDs of the gateways this device is assigned to.
+               description: The device IDs of the gateways that are registered to act on behalf of this device. Note that "via" and "memberOf" must not be set at the same time.
+            "viaGroups":
+               type: array
+               items:
+                  type: string
+               description: The IDs of the gateway groups that are registered to act on behalf of this device. Note that "viaGroups" and "memberOf" must not be set at the same time.
+            "memberOf":
+               type: array
+               items:
+                  type: string
+               description: The IDs of the gateway groups in which this device is a member. Note that neither "via" and "memberOf" nor "viaGroups" and "memberOf" must not be set at the same time.
```

Comment:
this looks like a new feature that a registry implementing this spec needs to support. FMPOV we should therefore bump the spec's minor version. WDYT @ctron @dejanb @calohmn

Reply:
@sophokles73 I agree. So that would be here:
https://github.com/eclipse/hono/blob/master/site/documentation/content/api/management/device-registry-v1.yaml#L39
(rebase needed first)

Reply:
Another thing regarding the description above: the double negation in the 2nd sentence sounds wrong. I'd suggest:
`Note that both "via" and "memberOf" as well as both "viaGroups" and "memberOf" must not be set at the same time.`

Reply:
@calohmn good point. I changed the formulation to two sentences.